tests/integration/components/tooltip-on-element-test.js

```diff
@@ -40,3 +40,18 @@ test('it has the proper aria-describedby tag', function(assert) {
   assert.equal(this.$(`#${describedBy}`).text().trim(), 'Some info in a tooltip.');
   assert.equal(describedBy.indexOf('#'), '-1');
 });
+
+test('it renders passed in classes', function(assert) {
```

Comment:
this is the exact type of test that we need, but can you add it to the `pass-through-test.js` file instead? You can just add classNames to the existing test and add more `.hasClass` calls